commands/user.go

```diff
@@ -616,18 +616,19 @@ func resetUserMfaCmdF(c client.Client, cmd *cobra.Command, args []string) error
 		return errors.New("expected at least one argument. See help text for details")
 	}
 
+	var result *multierror.Error
 	users, err := getUsersFromArgs(c, args)
 	if err != nil {
-		printer.PrintError(err.Error())
+		result = multierror.Append(result, err)
 	}
 
 	for _, user := range users {
 		if _, err := c.UpdateUserMfa(user.Id, "", false); err != nil {
-			printer.PrintError("Unable to reset user '" + user.Id + "' MFA. Error: " + err.Error())
+			result = multierror.Append(result, fmt.Errorf("Unable to reset user '"+user.Id+"' MFA. Error: "+err.Error()))
```

Comment:
You can use string interpolation here:
```suggestion
			result = multierror.Append(result, fmt.Errorf("Unable to reset user %q MFA. Error: %w", user.Id, err))
```